nix-ts-mode.el

```diff
@@ -193,14 +221,30 @@
     (setq-local treesit-font-lock-feature-list
                 '((comment builtin)
                   (keyword string path)
-                  (number constant attribute)
+                  (number constant attribute error)
                   (bracket delimiter operator ellipses function)))
 
+    ;; Comments
+    (setq-local comment-start "# ")
+    (setq-local comment-start-skip "#+\\s-*")
+
     ;; Indentation
+    (setq-local indent-tabs-mode nil)
```

Comment:
Is this a hard requirement for `nix` source? I don't think so. I'm in favour of spaces generally, but I don't think we should enforce it. Or does the indentation logic not work with tabs?

Reply:
I agree, I removed it.

Reply:
> Is this a hard requirement for nix source?

I am not sure about general nix files.  However, it is specified in [Nixpkgs manual](https://nixos.org/manual/nixpkgs/stable/#sec-syntax) and nix-mode [implements](https://github.com/NixOS/nix-mode/blob/719feb7868fb567ecfe5578f6119892c771ac5e5/nix-mode.el#L996-L1000) it.